### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by rngoode1**
Refs #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
...
 
-This will be a 2D Metroidvania style platformer.
-So it will focus on exploring, puzzlesolving and key hunting;
-with some light combat
-Current games similar to it are Hollow Knight and Celeste.
+Arachnivania is a 2D Metroidvania style platformer that will focus on exploring, puzzlesolving and key hunting with some light combat. Current games similar to it are Hollow Knight and Celeste.
+
+## Description
+
+In Arachnivania you play as a spider trying to build its web. In the process you will discover new ways to avoid predators and safe guard your home. This includes displaying warning colors or escaping by using silk.
```
Comment (RyanAScott) [inline README.md]: Will this be complicated to program? I'm still figuring out how to make a basic platformer level.

**Review by RyanAScott — CHANGES_REQUESTED** [review]
I say to plan out the gameplay a little better, the whole web thing sounds really complicated.

File: README.md (hunk 1 above)
Comment (rngoode1) [reply README.md]: I just wanted to get the basic structure of the doc complete. I just put something up there. We can definitely go over the detailed description.

Comment (rngoode1) [reply README.md]: Revised doc to include updated description and story.

**Comment by rngoode1** [comment]
Updated Readme doc

**Review by RyanAScott — APPROVED** [review]

**Pull request merged**